styles/designs/carnival/pdf/_page-breaks.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../common/pdf/page-breaks';
+
+// EOC exercises in double columns shouldn't be broken up
+
+.os-eoc {
+  > .ap-test-prep > [data-type="exercise"] {
+    page-break-inside: auto;
+  }
+}
```

Comment:
@LostMojo There is a rule overwriting including simmilar styling in u-physics book.scss file: 
![image](https://user-images.githubusercontent.com/55203598/175608013-8fae0645-789b-4b51-b79e-299531b887bc.png)
I guess this rule should be added to all exercises within 2 clumns in carnival? But maybe since this is urgent you could open a separate issue about it?  What do you think?

Reply:
I think I can give it a shot today to see if it'll make things prettier, and I can push this fix and open a new card if it doesn't-- this should hit all the eoc exercises, so we wouldn't need to target 2 column exercises specifically, I don't think

Reply:
The way the two column modules are targeted is through their classes (some of which exist in other carnival books) so this might need to be a book-by-book fix (or an optional _page-break.scss shape?)

Reply:
Yes, I guess we should only target specific two columns sections by it's classes.One option is to do this is do it book by book :+1:  as you wrote, becuase we need to overwrite this rule I suppose: 
![image](https://user-images.githubusercontent.com/55203598/175611761-7caaa54d-1f57-4ded-be29-b7658ca230f0.png)
Another easier option is to exclude specific classes from the above rule e.g.:
```
.os-eoc:not(.ap-test-prep) {
  > section > [data-type="exercise"] {
page break rule
}
```

Reply:
<img width="516" alt="Screen Shot 2022-06-24 at 1 04 50 PM" src="https://user-images.githubusercontent.com/5740972/175617764-f426014a-4667-4623-b939-20f6d04532fc.png">

Reply:
This does look better!